src/main/java/seedu/ultron/logic/parser/AddCommandParser.java

```diff
@@ -63,5 +81,21 @@ public AddCommand parse(String args) throws ParseException {
     private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
+    private static boolean arePrefixesAbsent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```

Comment:
Duplicate method

Reply:
It's not duplicate, the latter is checking whether all the fields are absent and the former is checking whether there are any absent